### DISCUSSION TITLE
New option to use Click-House on browser :)

### DISCUSSION THREAD

**Discussion by caioricciuti** [ClickHouse/ClickHouse#63996] [Show and tell]
Hello all, I've been working on a UI for click-house for myself for a while, the project looks cool (of course I think that hahaha). Anyway since I don't know any other easy tool to connect to Click House other than TABIX I've made one, it's not perfect but it serves my purpose on a daily basis, for anyone who wants to test, support, and if possible start, please go ahead! Thank you all very much 

https://github.com/caioricciuti/ch-ui

note, to run in docker:
``` bash
docker run -p 5521:5521 ghcr.io/caioricciuti/ch-ui:latest
```

Hope it helps the community! :)

**Comment by tbragin** [comment]
Thank you for sharing! Is it primarily aimed to serve as building a SQL console like UI (for analyst and admin), or custom UI for displaying data for external users (e.g. dashboards for consumers)?

**Reply by caioricciuti** [reply]
Hello @tbragin, it's was created to be a admin, analyst tool... The only query output is a table or json format, you can download the query result as csv/json but there is no option to create dashboards. Thank you for asking.
